mkosi

```diff
@@ -2287,7 +2287,7 @@ def load_args():
             if platform.machine() == "aarch64":
                 args.mirror = "http://mirror.archlinuxarm.org"
         elif args.distribution == Distribution.opensuse:
-            args.mirror = "https://download.opensuse.org"
+            args.mirror = "http://download.opensuse.org"
```

Comment:
https works fine on Fedora: c.f. #6231. What error are you getting?

Reply:
@fbuihuu See this thread for an explanation why this does not work: https://forums.opensuse.org/showthread.php/527052-Cached-files-in-home-cache-kioexec-krun-_-How-to-avoid?p=2838516#post2838516

Reply:
@fbuihuu And also see this very detailed post: https://progress.opensuse.org/news/40